osbs/build/build_request.py

```diff
@@ -99,6 +99,7 @@ def set_params(self, **kwargs):
         :param auto_build_node_selector: dict, a nodeselector for auto builds
         :param isolated_build_node_selector: dict, a nodeselector for isolated builds
         :param is_auto: bool, indicates if build is auto build
+        :param parent_images_digests: dict, mapping image digests to names and platforms
```

Comment:
What exactly is the structure of this? What are the keys, and what are the values?

Reply:
Reworded